iaso/migrations/0237_add_modules_to_all_existing_accounts.py

```diff
@@ -7,7 +7,8 @@
 
 def add_modules_to_accounts(apps, schema_editor):
     modules = [module["codename"] for module in MODULES]
-    all_accounts = Account.objects.all()
+    account = apps.get_model("iaso", "Account")
+    all_accounts = account.objects.all()
```

Comment:
Why is this in this PR?

Reply:
It's because, the migration was generating an error that the field analytics_script didn't exist. So I request the model Account before the field to be added to fix the error.

Reply:
I think this was only necessary because you were in a strange migration state (having the field analytics_script in the db already due to a previous migration then running that migration). Do you think that's possible?

Reply:
First I made a migration to remove the analytics_script from the Page model **iaso/migrations/0252_remove_analytics_script_field_from_page.py**, after I added the analytics_script to the Account model **iaso/migrations/0253_add_analytics_script_field_into_account.py** and in local everything worked fine. It's when I pushed on github that the migration precisely on this migration and on the line **all_accounts = Account.objects.all()** saying that **analytics_script does not exist yet**.